docs/src/pages/components/slider/VerticalAccessibleSlider.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/core/Slider';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  forceWebkitVerticalOrientation: {
+    '& input[type="range"]': {
+      '-webkitAppearance': 'slider-vertical',
+    },
+  },
+});
+
+export default function VerticalSlider() {
+  const classes = useStyles();
+
+  function preventHorizontalKeyboardNavigation(event: React.KeyboardEvent) {
+    if (event.key === 'ArrowLeft' || event.key === 'ArrowRight') {
+      event.preventDefault();
+    }
+  }
+
+  return (
+    <React.Fragment>
+      <Typography id="vertical-accessible-slider" gutterBottom>
+        Temperature
+      </Typography>
+      <Box sx={{ height: 300 }}>
+        <Slider
+          className={classes.forceWebkitVerticalOrientation}
```

Comment:
I think that it's better to avoid using the legacy style API. `sx` could be simple enough and might be short enough to be visible in the inline preview. Otherwise, we can use styled().

```suggestion
          sx={{
            '& input[type="range"]': {
              WebkitAppearance: 'slider-vertical',
            },
          }}
```

Reply:
Which we can revisit once it's integrated.

Reply:
What do you mean by integrated? The two proposed solutions are working.

Reply:
>  plan to deprecate them in v5.

They're not deprecated yet.

Reply:
> They're not deprecated yet.

Please don't use withStyles/makeStyles for any new code. It's legacy at this point.